ewatercycle_experiment_launcher/openapi.yaml

```diff
@@ -225,141 +161,62 @@ components:
-            config:
-              type: object
-              additionalProperties: false
-              required:
-                - format
-                - url
-              properties:
-                format:
-                  type: string
-                  example: ini
-                url:
-                  type: string
-                  example: >-
-                    https://raw.githubusercontent.com/UU-Hydro/PCR-GLOBWB_input_example/master/RhineMeuse30min/ini_and_batch_files/rapid/setup_natural_test.ini
-            plotting:
-              type: object
-              additionalProperties: false
-              required:
-                - variable
-                - index
-              properties:
-                variable:
-                  type: string
-                  example: discharge
-                index:
-                  type: number
-                  example: 86
-    AssessmentRequest:
-      type: object
-      required:
-        - notebook
-        - setup
-      additionalProperties: false
-      properties:
-        notebook:
-          $ref: '#/components/schemas/NotebookRequest'
-        setup:
-          type: object
-          required:
-            - model
-            - datafiles
-            - config
-            - assessment
+            - variable
+            - observation
           additionalProperties: false
           properties:
             model:
               type: object
               required:
                 - name
-                - grpc4bmi_container
+                - version
               properties:
                 name:
                   type: string
-                  description: Name of model
-                  example: PCR-GLOBWB
-                grpc4bmi_container:
+                  description: Name of model as defined in ewatercycle.models module
+                  example: PCRGlobWB
+                version:
                   type: string
-                  example: 'ewatercycle/pcrg-grpc4bmi:latest'
-            datafiles:
-              type: object
+                  description: Version of model to use
+                  example: setters
+                lumped:
+                  type: boolean
+                  description: Whether model is lumped or non-lumped model
+                  default: false
               additionalProperties: false
-              required:
-                - format
-                - url
-              properties:
-                format:
-                  type: string
-                  example: svn
-                url:
-                  type: string
-                  example: >-
-                    https://github.com/UU-Hydro/PCR-GLOBWB_input_example/trunk/RhineMeuse30min
-            config:
+            forcing:
+                type: string
+                description: Directory name of forcing. Directory should contain `ewatercycle_forcing.yaml` file.
+                example: marrmot-m01_example_1989-1992_buffalo-river
```

Comment:
It would indeed be nice to generate forcings in notebook, for now I created https://github.com/eWaterCycle/experiment-launcher/issues/30 to add it later.